.pre-commit-config.yaml

```diff
@@ -42,7 +42,11 @@ repos:
     rev: v2.2.2
     hooks:
     -   id: codespell
-        types_or: [python, rst, markdown]
+        types_or: [python, rst, markdown, cython, c]
+        exclude: |
+            (?x)
+            |pandas/_libs/ops_dispatch.pyx
```

Comment:
why are these excluded?

Reply:
<img width="456" alt="image" src="https://user-images.githubusercontent.com/77875500/223100709-792bd4be-adea-4ab5-b7ed-55d459380f71.png">

`nin` cause error

Reply:
should I add some comment to explain them?

Reply:
thanks - probably fine to add both to

https://github.com/pandas-dev/pandas/blob/4ab1c76a4a28aca442410da173f91c42caa7c915/pyproject.toml#L610

then